drivers/base/regmap/regmap-sdw-mbq.c

```diff
@@ -98,4 +98,4 @@ struct regmap *__devm_regmap_init_sdw_mbq(struct sdw_slave *sdw,
 EXPORT_SYMBOL_GPL(__devm_regmap_init_sdw_mbq);
 
 MODULE_DESCRIPTION("Regmap SoundWire MBQ Module");
-MODULE_LICENSE("GPL v2");
+MODULE_LICENSE("GPL");
```

Comment:
the commit message could be misleading. We are not changing the license at all.

use MODULE_LICENSE("GPL")

also add a pointer to the licensing rules for reference.